components/Bill/Bill.js

```diff
@@ -35,7 +36,10 @@ const ViewBillPage = props => {
       </div>
       <div>
         {bill && bill.DocumentText != null
-          ? bill.DocumentText.substring(1, 700) + "..."
+          ? <>
+          <span> {bill.DocumentText.substring(1, 700) + "..." } </span>
```

Comment:
We can use [`white-space: pre-wrap`](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space) to preserve the tabs and newlines in the text. Also we should start the substring at 0 to include the first character:

```
            <span style={{ whiteSpace: "pre-wrap" }}>
              {bill.DocumentText.substring(0, 700) + "..."}
            </span>
```